BridgeBidder/Conventions/Gerber.cs

```diff
@@ -10,9 +10,10 @@ public class Gerber : Bidder
 	//	private static (int, int) SmallSlam = (31, 35);
 		private static (int, int) GrandSlam = (36, 100);
 
-		public static StaticConstraint Applies = new StaticConstraint((call, ps) => 
+		public static StaticConstraint Applies = new SimpleStaticConstraint((call, ps) => 
 				ps.Partner.Bid is Bid partnerBid &&
-				partnerBid.Strain == Strain.NoTrump && partnerBid.Level < 3);
+				partnerBid.Strain == Strain.NoTrump && partnerBid.Level < 3, 
+				description: "partner has bid 1NT or 2NT");
```

Comment:
Can we make this `GetLogDescription` only? Probably requires enhancing `SimpleStaticConstraint` slightly to add a new optional `logDescription` param.

In general, I'd like omit bits from the description that just repeat what another player bid.